src/store/playerActions.js

```diff
@@ -59,11 +65,43 @@ export const playWord = (word) => ({
     word
 });
 
+export const voteForCardBegin = () => ({
+    type: types.VOTE_FOR_CARD_BEGIN
+});
+
+export const voteForCardSuccess = (card) => ({
+    type: types.VOTE_FOR_CARD_SUCCESS,
+    payload: { card }
+});
+
+export const voteForCardFailure = (error) => ({
+    type: types.VOTE_FOR_CARD_FAILURE,
+    payload: { error }
+});
+
+export const voteForCard = (id) => {
+    return (dispatch) => {
+        dispatch(voteForCardBegin());
+        return axios.post('/api/voteCard', {
```

Comment:
Two points here:
- Do you need to return the `axios` request?
- If you want to clean up thunks, you can do something like this
```
export const voteForCard = id => dispatch => {
  // Function
}
```